src/logic/addressBook/utils/__tests__/addressBookUtils.test.ts

```diff
@@ -199,3 +211,102 @@ describe('getAddressBookFromStorage', () => {
     expect(spy).toHaveBeenCalled()
   })
 })
+
+describe('isValidAddressBookName', () => {
+  it('It should return false if given a blacklisted name like UNKNOWN', () => {
+    // given
+    const addressNameInput = 'UNKNOWN'
+
+    const expectedResult = false
+
+    // when
+    const result = isValidAddressBookName(addressNameInput)
+
+    // then
+    expect(result).toStrictEqual(expectedResult)
+  })
+  it('It should return false if given a blacklisted name like MY WALLET', () => {
+    // given
+    const addressNameInput = 'MY WALLET'
+
+    const expectedResult = false
+
+    // when
+    const result = isValidAddressBookName(addressNameInput)
+
+    // then
+    expect(result).toStrictEqual(expectedResult)
+  })
+  it('It should return false if given a blacklisted name like OWNER #', () => {
+    // given
+    const addressNameInput = 'MY WALLET'
+
+    const expectedResult = false
+
+    // when
+    const result = isValidAddressBookName(addressNameInput)
+
+    // then
+    expect(result).toStrictEqual(expectedResult)
+  })
+  it('It should true false if the given address is not blacklisted', () => {
+    // given
+    const addressNameInput = 'User'
+
+    const expectedResult = true
+
+    // when
+    const result = isValidAddressBookName(addressNameInput)
+
+    // then
+    expect(result).toEqual(expectedResult)
+  })
+})
+
+describe('checkIfEntryWasDeletedFromAddressBook', () => {
+  const mockAdd1 = '0x696fd93D725d84acfFf6c62a1fe8C94E1c9E934A'
+  const mockAdd2 = '0x2C7aC78b01Be0FC66AD29b684ffAb0C93B381D00'
+  const mockAdd3 = '0x537BD452c3505FC07bA242E437bD29D4E1DC9126'
+  const entry1 = getMockAddressBookEntry(mockAdd1, 'test1')
+  const entry2 = getMockAddressBookEntry(mockAdd2, 'test2')
+  const entry3 = getMockAddressBookEntry(mockAdd3, 'test3')
+  it('It should return true if given an entry was deleted from addressBook', () => {
```

Comment:
Correct title

`It should return true if a given entry was deleted from addressBook`